neotools/file.py

```diff
@@ -249,16 +249,14 @@ def create_file(device, filename, password, data, applet_id):
     :param device:
     :param filename:
     :param password:
-    :param data: The buffer to write.
+    :param text: The raw buffer to write.
```

Comment:
I saw some line break characters when creating a new file, though I attributed it to something about the AS3000. I'll check on a Neo in a bit. (And actually, it might not have been a new file anyway; for this I think it has to be a named file beyond the first 8.)

Reply:
Concerning `data` and `text`... I was trying to match corresponding usage in `commands` module: `read_all_files`, `read_text`, `read_file`, `write_file`, etc. However, that's not the convention in this file. I will change it back.

Reply:
I intended `commands.write_file` to be used only for AlphaWord, so I called the contents text. With your change, when it can be used for any applet, this is no longer the case. For example, the custom dictionary in Thesaurus is not plain text, as far as I remember.

But file.py is a low-level module that shouldn't assume that it is dealing with any particular applet. So I'd like to keep calling it `data`. It is a nice change to remove the hacky conditional utf8 encoding and change from "buffer" to "raw buffer" in the docstring.

Reply:
I tried with some German text, and it looks like creating files doesn't work either before or after this change. I am looking into it and will resolve if it is straightforward.

Reply:
I must've had an old version installed. The new version seems to write new files properly whereas the old did not (due to the UTF-8 conversion). Test was as below:
```neotools files write test test1 #new
neotools files write test test2 #new
neotools files write test test2 #overwrite
neotools files write test 1 #overwrite```
All files appear correct and the same as each other on the Neo.